packages/web-app-search/src/views/List.vue

```diff
@@ -10,7 +10,7 @@ import { debounce } from 'lodash-es'
 
 export default defineComponent({
   data() {
-    const { provider: providerId } = this.$route.query
+    const { provider: providerId } = (this.$route as any).query
```

Comment:
why don't we cast `.query` instead?

Reply:
For some reason vue-tsc thinks `this.$route` is a Function ... I can live with this, we should just port this component to composition api as soon as we touch it again